### PR TITLE
Added Todd Profile

### PR DESCRIPTION
**Pull request by 05tb**
Add my profile page. 

The mugshot doesn't go anywhere but is ready for when we get the new images.

### CODE REVIEW

**Comment by 05tb** [comment]
My profile page.
It needs an image linked to it but thought I'd wait for the new ones.

File: _profiles/todd-burlington.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: Todd Burlington
+layout: profile
+name: Todd Burlington
+summary: An undergraduate physics student turn summer lab rat.
+affiliation: core
```
Comment (niallrobinson) [inline _profiles/todd-burlington.md]: not core - haven't we got a intern affiliation or something?

Comment (05tb) [inline _profiles/todd-burlington.md]: I wasn't sure what to add it under. I believe @jacobtomlinson mentioned something about a sort of lab alumni page (?), thats what I was a aiming for with this profile.

Comment (jacobtomlinson) [inline _profiles/todd-burlington.md]: Currently there are two affiliations `core` and `past-member`. It is a right of passage to merge the PR to change yourself to a `past-member` as your last duty in the lab. 

Maybe there should also be an `intern` and `past-intern` affiliation. But core is fine for now.

**Comment by jacobtomlinson** [comment]
Your profile pic is now available here https://images.informaticslab.co.uk/profiles/cropped/todd-burlington.png, please update the PR.

**Comment by jacobtomlinson** [comment]
Failure due to #349

**Comment by jacobtomlinson** [comment]
#348 changes the way affiliations work. You can change your affiliation to `intern` and also add a new property called `current` and set it true, then we'll change to to false next week.

```yaml
affiliation: intern
current: true
```

**Comment by jacobtomlinson** [comment]
Once #350 has been merged you can rebase this onto the end of master and the tests should pass

**Comment by jacobtomlinson** [comment]
Just in case you haven't come across rebasing before.

 * Checkout master
 * do a `git pull`
 * Checkout your branch for this PR
 * run `git rebase master`
 * `git push`

**Comment by jacobtomlinson** [comment]
FYI This has all the rhino stuff in it too.

**Comment by 05tb** [comment]
History seems to have gotten mixed up. 

Closed this PR, will open a new PR for my profile.

**Pull request closed without merging**